pkg/cmd/pulumi/stack_history.go

```diff
@@ -162,7 +163,9 @@ func displayUpdatesJSON(updates []backend.UpdateInfo, decrypter config.Decrypter
 			info.EndTime = makeStringRef(time.Unix(update.EndTime, 0).UTC().Format(timeFormat))
 			resourceChanges := make(map[string]int)
 			for k, v := range update.ResourceChanges {
-				resourceChanges[string(k)] = v
+				if k != deploy.OpOutputChange {
```

Comment:
Why do we filter out OpOutputChange here? That's not obvious from context, deserves an explanation comment.

Reply:
Added a comment